modules/identity-provider-configuring-apache-request-header.adoc

```diff
@@ -1,104 +1,75 @@
 // Module included in the following assemblies:
 //
-// * orphaned
+// * authentication/identity_providers/configuring-request-header-identity-provider.adoc
 
 [id="identity-provider-configuring-apache-request-header_{context}"]
 = Configuring Apache authentication using request header
 
 This example configures an authentication proxy on the same host as the master.
-Having the proxy and master on the same host is merely a convenience and may not
+Having the proxy and master on the same host is merely a convenience and might not
```

Comment:
@stlaz I'm attempting to piece together the history of this section of content. From what I've gathered, in 3.x, we were required to interact with the master because it was the only way users could authenticate with the OAuth server. In 4.x, it seems working through the master to authenticate with the OAuth server is no longer necessary. Instead, the namespace’s route is the gateway to accessing the OAuth server. Is that correct?

With that said, should the theme of this paragraph (and module) simply be that we're configuring an Apache authentication proxy for the OpenShift Container Platform? For example:

> This example configures an Apache authentication proxy for the OpenShift Container Platform using the request header identity provider. Having the proxy and an OpenShift Container Platform route on the same host is merely a convenience and might not be suitable for your environment. For example, if you were already running a router on OpenShift Container Platform, port 443 would be unavailable.

Is that close? Or are there any edits that would clarify our intentions?

Just trying to get my head wrapped around the purpose here, which will make it easier to update the remaining content. Thanks!

Reply:
@stlaz thanks for the explanation. That makes it clearer what we're doing in this example. I'll make some finishing touches and let you know when it's ready for a (hopefully :crossed_fingers:) final review. Thanks!